pyinfrabox/utils.py

```diff
@@ -22,21 +22,29 @@ def check_text(t, path, allowEmpty=False):
     if not allowEmpty and not t:
         raise ValidationError(path, "empty string not allowed")
 
+
 def check_allowed_properties(d, path, allowed):
     if not isinstance(d, dict):
         raise ValidationError(path, "must be an object")
 
     for key in d:
         if key not in allowed:
-            raise ValidationError(path, "invalid property '%s'" % key)
+            if 'name' in d.keys():
+                raise ValidationError(path + '(' + d['name'] + ')', "invalid property '%s'" % key)
```

Comment:
For readability
```python
'%s(%s)' % (path, d['name])
```
(same on line 45)

Reply:
Added the check for pyyaml version;
Re-format the message output;